packages/components/tests/dummy/app/templates/components/form/base-elements.hbs

```diff
@@ -576,4 +576,23 @@
     {{/let}}
   </div>
 
+  {{! ###################### }}
+
+  <Shw::Divider />
+
+  <Shw::Text::H2>Visibility toggle</Shw::Text::H2>
+
+  <Shw::Grid @columns={{4}} {{style width="fit-content"}} as |SG|>
```

Comment:
you can probably use a flex container here, so no need to set the width and the columns